### PR TITLE
Removed redundant SubcomposeLayout in favor of Layout

### PR DESCRIPTION
**Pull request by RareScrap**
Fix of #23 and #11

### CODE REVIEW

**Comment by onebone** [comment]
Sorry for the review being late, I think it will take some time. I will do it as soon as possible.

**Review by onebone — CHANGES_REQUESTED** [review]

File: lib/src/main/java/me/onebone/toolbar/CollapsingToolbarScaffold.kt [hunk 1]
```diff
@@ -124,34 +123,26 @@ fun CollapsingToolbarScaffold(
...
 
 		layout(width, height) {
+			toolbarPlaceable.place(0, state.offsetY)
 			bodyPlaceables.forEach {
 				it.place(0, toolbarHeight + state.offsetY)
 			}
-
-			toolbarPlaceables.forEach {
-				it.place(0, state.offsetY)
-			}
```
[the remark this answers is not in the document]
Comment (RareScrap) [reply lib/src/main/java/me/onebone/toolbar/CollapsingToolbarScaffold.kt:150]: I did just that - moved the toolbar to the top

File: lib/src/main/java/me/onebone/toolbar/CollapsingToolbarScaffold.kt [hunk 2]
```diff
@@ -124,34 +123,26 @@ fun CollapsingToolbarScaffold(
 			}
 		)
 
-		val bodyPlaceables = subcompose(CollapsingToolbarScaffoldContent.Body) {
-			body()
-		}.map { it.measure(bodyConstraints) }
+		val toolbarPlaceable = measurables[0].measure(toolbarConstraints)
+		val bodyPlaceables =
+			measurables.drop(1).map { it.measure(bodyConstraints) }// [1].measure(bodyConstraints)
```
[the remark this answers is not in the document]
Comment (RareScrap) [reply lib/src/main/java/me/onebone/toolbar/CollapsingToolbarScaffold.kt:128]: No, I just forgot to put it away. Fixed

File: lib/src/main/java/me/onebone/toolbar/CollapsingToolbarScaffold.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (onebone) [reply lib/src/main/java/me/onebone/toolbar/CollapsingToolbarScaffold.kt:150]: Sorry for the confusion, what I meant by _top_ was from the layout perspective. Compose puts the first one to the bottom like a stack.

image
image

**Review by onebone — CHANGES_REQUESTED** [review]

File: lib/src/main/java/me/onebone/toolbar/CollapsingToolbarScaffold.kt (hunk 2 above)
[the remark this answers is not in the document]
Comment (onebone) [reply lib/src/main/java/me/onebone/toolbar/CollapsingToolbarScaffold.kt]: Sorry for the confusion, what I meant by _top_ was from the layout perspective. Compose puts the first one to the bottom like a stack.

image
image

File: lib/src/main/java/me/onebone/toolbar/CollapsingToolbarScaffold.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (onebone) [reply lib/src/main/java/me/onebone/toolbar/CollapsingToolbarScaffold.kt:150]: Sorry for the confusion, what I meant by _top_ was from the layout perspective. Compose puts the first one to the bottom like a stack.

image
image

[the remark this answers is not in the document]
Comment (RareScrap) [reply lib/src/main/java/me/onebone/toolbar/CollapsingToolbarScaffold.kt]: Fixed. Thank you for notice this.

[the remark this answers is not in the document]
Comment (toddrandy79) [reply lib/src/main/java/me/onebone/toolbar/CollapsingToolbarScaffold.kt]: Good job

**Review by onebone — APPROVED** [review]
Thank you for your contribution! I was actually quite confused about what SubcomposeLayout was for and thought of it as a graceful way to create a slot in layout. I appreciate your hard research work and this contribution for making me clear of the concept 😃 😃 👍 👍

**Pull request merged**